pkg/api/v1/atlascluster_types_test.go

```diff
@@ -0,0 +1,117 @@
+package v1
+
+import (
+	"reflect"
+	"sort"
+	"testing"
+
+	"github.com/fatih/structtag"
+	"github.com/stretchr/testify/assert"
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+var excludedClusterFieldsOurs = map[string]bool{}
+var excludedClusterFieldsTheirs = map[string]bool{}
+
+func init() {
+	excludedClusterFieldsOurs["projectRef"] = true
+
+	excludedClusterFieldsTheirs["backupEnabled"] = true
+	excludedClusterFieldsTheirs["id"] = true
+	excludedClusterFieldsTheirs["groupId"] = true
+
+	// Deprecated
+	excludedClusterFieldsTheirs["replicationSpec"] = true
+	excludedClusterFieldsTheirs["replicationFactor"] = true
+
+	// These fields are shown in the status
+	excludedClusterFieldsTheirs["mongoDBVersion"] = true
+	excludedClusterFieldsTheirs["mongoURI"] = true
+	excludedClusterFieldsTheirs["mongoURIUpdated"] = true
+	excludedClusterFieldsTheirs["mongoURIWithOptions"] = true
+	excludedClusterFieldsTheirs["connectionStrings"] = true
+	excludedClusterFieldsTheirs["srvAddress"] = true
+	excludedClusterFieldsTheirs["stateName"] = true
+
+	// CLOUDP-80765
+	excludedClusterFieldsTheirs["paused"] = true
+
+	// TODO why not included?
```

Comment:
This was implemented [11 days ago](https://github.com/mongodb/go-client-mongodb-atlas/commit/41343eb93a4848048a0e5a6f6d98dc421cecc3c2#diff-cdec3370b78f74b0e08291a906d41ff1be66a9a9c0f7a5d3cfbbf9fe7ad23e2eR33) - our spec is out of date already 🙂

Reply:
wow, nice! Seems like the test is working :) 
I'll add the field in this PR